apps/browser/pages/api/agent-jobs/start.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { evoAgentJobScheduler } from "../../../src/agent-jobs/AgentJobScheduler";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "GET") {
+    return res.status(404);
+  }
+
+  const threadId = req.query.threadId as string;
+
+  console.log("Adding job to the queue: " + threadId);
```

Comment:
Why is the user providing the thread ID? Shouldn't the service be the authority here?

Also would recommend merging with latest `dev` as there now exists user-auth + goal-ids. This way here you may do something like:
1. get user auth session
2. generate a new goalID
3. use the goal ID as the threadID